greymatter-io/operator
# Add robust update script based on gm-tools/scripts/update

## How to test

Run `scripts/update` from anywhere, and it should do all of the steps necessary to update the `main` branch of the operator, and then return you to your original state, whether or not files were changed, and whether or not you were on `main` to begin with.
